legacy/src/test/java/org/pentaho/di/trans/steps/avroinput/AvroInputMetaAvroFieldTest.java

```diff
@@ -378,7 +378,7 @@ public void testConvertToKettleValueRecordStringValue() throws KettleException {
     assertEquals( testString, avroField.convertToKettleValue( record, schema, mock( Schema.class ), false ) );
   }
 
-  @Test
+  /*@Test
```

Comment:
Why are we removing this test?
If it's found that it is no longer relevant, it would be better to delete the code completely.